README.md

```diff
@@ -237,6 +249,9 @@ Adjust the port to your liking, but make sure that you change both numbers (the
 ...
 ```
 
+Now that your port changes are set you have to recreate your container. Therefore you need to use `docker-compose up -d` in order to apply your port changes.
```

Comment:
I think a comma is missing after `set`.

Reply:
addressed with [45e3cfc](https://github.com/mschnitzer/ark-survival-ascended-linux-container-image/pull/82/commits/45e3cfc808e52bdf1f1af888a395b8a238ddf458)